Frontend/src/app/core/services/project.service.ts

```diff
@@ -33,6 +33,10 @@ export class ProjectService {
     return this.http.post(`${this.api}/Project`, projectCreateInfo);
   }
 
+  public updateProjectDetails(description: string, projectId: number) {
+    return this.http.patch(`${this.api}/Project`, { description, projectId })
```

Comment:
I feel like for sending more then 1 item we should be mirroring the DTO object as an interface, just my preference I think it makes it easier to read

Reply:
fixed